iOS_SDK/OneSignalSDK/Source/OneSignalCommonDefines.h

```diff
@@ -66,8 +66,8 @@
 #define OSUD_PLAYER_ID_FROM                                                 @"GT_PLAYER_ID_LAST"                                                // * OSUD_PLAYER_ID_FROM
 #define OSUD_PUSH_TOKEN_TO                                                  @"GT_DEVICE_TOKEN"                                                  // * OSUD_PUSH_TOKEN_TO
 #define OSUD_PUSH_TOKEN_FROM                                                @"GT_DEVICE_TOKEN_LAST"                                             // * OSUD_PUSH_TOKEN_FROM
-#define OSUD_USER_SUBSCRIPTION_TO                                           @"ONESIGNAL_SUBSCRIPTION"                                           // * OSUD_USER_SUBSCRIPTION_TO
-#define OSUD_USER_SUBSCRIPTION_FROM                                         @"ONESIGNAL_SUBSCRIPTION_SETTING"                                   // * OSUD_USER_SUBSCRIPTION_FROM
+#define OSUD_PUSH_DISABLED_TO                                               @"ONESIGNAL_PUSH_DISABLED"                                          // * OSUD_PUSH_DISABLED_TO
```

Comment:
I think we want to keep the same names for UserDefaults, to avoid loosing saved data

Reply:
Fixed!